src/main/java/Db/dbConfig.java

```diff
@@ -0,0 +1,25 @@
+package Db;
+
+import javax.sql.DataSource;
+import org.apache.commons.dbcp2.BasicDataSource;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+
+@Configuration
```

Comment:
with spring boot you could do all that configuration in application.properties/application.yml